MatchError while using MultiClockCoreplex

I try to use `MultiClockCoreplex` at the current master branch (commit 549e006988). First, I have to modify code as below. Then I have to add extra I/O ports for different clock and reset input in the `RocketExampleTop` module, and connect them to the rocket tiles.

```diff
--- a/src/main/scala/rocketchip/RocketPlexMaster.scala
+++ b/src/main/scala/rocketchip/RocketPlexMaster.scala
@@ -14,7 +14,7 @@ trait RocketPlexMaster extends L2Crossbar {
   val module: RocketPlexMasterModule
   val mem: Seq[TLInwardNode]
 
-  val coreplex = LazyModule(new DefaultCoreplex)
+  val coreplex = LazyModule(new MultiClockCoreplex)
 
   coreplex.l2in := l2.node
   socBus.node := coreplex.mmio
```

But when I do the modification as the diff shows above, I get the following error.

```
[info] Set current project to rocketchip (in build file:/home/yuzihao/riscv_pard_fpga/rocket-chip/)
[info] Compiling 1 Scala source to /home/yuzihao/riscv_pard_fpga/rocket-chip/chisel3/target/scala-2.11/classes...
[info] Compiling 8 Scala sources to /home/yuzihao/riscv_pard_fpga/rocket-chip/target/scala-2.11/classes...
[warn] there were 7 feature warnings; re-run with -feature for details
[warn] one warning found
[info] Compiling 169 Scala sources to /home/yuzihao/riscv_pard_fpga/rocket-chip/target/scala-2.11/classes...
[warn] there were 3102 feature warnings; re-run with -feature for details
[warn] one warning found
[info] Running rocketchip.Generator /home/yuzihao/riscv_pard_fpga/rocket-chip/vsim/generated-src rocketchip TestHarness rocketchip DefaultConfig
[info] [0.003] Elaborating design...

Interrupt map:
  [1, 2] => ExampleRocketTop
[error] (run-main-0) java.lang.reflect.InvocationTargetException
java.lang.reflect.InvocationTargetException
        at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
        at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
        at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
        at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
        at util.HasGeneratorUtilities$$anonfun$2.apply(GeneratorUtils.scala:47)
        at util.HasGeneratorUtilities$$anonfun$2.apply(GeneratorUtils.scala:44)
        at chisel3.core.Module$.do_apply(Module.scala:29)
        at chisel3.Driver$$anonfun$elaborate$1.apply(Driver.scala:190)
        at chisel3.Driver$$anonfun$elaborate$1.apply(Driver.scala:190)
        at chisel3.internal.Builder$$anonfun$build$1.apply(Builder.scala:194)
        at chisel3.internal.Builder$$anonfun$build$1.apply(Builder.scala:192)
        at scala.util.DynamicVariable.withValue(DynamicVariable.scala:58)
        at chisel3.internal.Builder$.build(Builder.scala:192)
        at chisel3.Driver$.elaborate(Driver.scala:190)
Caused by: scala.MatchError: TLCacheEdge (of class rocket.TLCacheEdge$)
        at config.TerminalView.find(Config.scala:52)
        at config.EmptyParameters.chain(Config.scala:64)
        at config.ChainView.find(Config.scala:56)
        at config.FunctionParameters.chain(Config.scala:69)
        at config.Config.chain(Config.scala:41)
        at config.ChainView.find(Config.scala:56)
        at config.FunctionParameters.chain(Config.scala:69)
        at config.Config.chain(Config.scala:41)
        at config.ChainParameters.chain(Config.scala:60)
        at config.Config.chain(Config.scala:41)
        at config.ChainView.find(Config.scala:56)
        at config.FunctionParameters.chain(Config.scala:69)
        at config.Config.chain(Config.scala:41)
        at config.ChainParameters.chain(Config.scala:60)
java.lang.RuntimeException: Nonzero exit code: 1
        at scala.sys.package$.error(package.scala:27)
        at sbt.BuildCommon$$anonfun$toError$1.apply(Defaults.scala:2077)
        at sbt.BuildCommon$$anonfun$toError$1.apply(Defaults.scala:2077)
        at scala.Option.foreach(Option.scala:236)
        at sbt.BuildCommon$class.toError(Defaults.scala:2077)
        at sbt.Defaults$.toError(Defaults.scala:39)
        at sbt.Defaults$$anonfun$runMainTask$1$$anonfun$apply$36$$anonfun$apply$37.apply(Defaults.scala:740)
        at sbt.Defaults$$anonfun$runMainTask$1$$anonfun$apply$36$$anonfun$apply$37.apply(Defaults.scala:738)
        at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
        at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:40)
        at sbt.std.Transform$$anon$4.work(System.scala:63)
        at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
        at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
        at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
[error] (rocketchip/compile:runMain) Nonzero exit code: 1
[error] Total time: 118 s, completed Nov 25, 2016 10:37:04 AM
```

--- a/src/main/scala/coreplex/RocketTiles.scala
+++ b/src/main/scala/coreplex/RocketTiles.scala
@@ -57,7 +57,7 @@ class AsyncRocketTile(tileId: Int)(implicit p: Parameters) extends LazyModule {
       val uncached = uncachedOut.bundleOut
       val slave = slaveNode.map(_.bundleIn)
       val hartid = UInt(INPUT, p(XLen))
-      val interrupts = new TileInterrupts().asInput
+      val interrupts = new TileInterrupts()(rocket.coreParams).asInput
       val resetVector = UInt(INPUT, p(XLen))
     }
     rocket.module.io.interrupts := ShiftRegister(io.interrupts, 3)

